File: JavaScript/1035.js
/*
Leia 4 valores inteiros A, B, C e D. A seguir, se B for maior do que C e se D for maior do que A, e a soma de C com D for maior que a soma de A e B e se C e D, ambos, 
forem positivos e se a variável A for par escrever a mensagem "Valores aceitos", senão escrever "Valores nao aceitos".
*/
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => +a);
let [A,B,C,D] = pegarValores(lines.shift())

if(B>C && D>A && (C+D)>(A+B) && C > 0 && D > 0 && A % 2 === 0){
    console.log("Valores aceitos")
}else{
    console.log("Valores nao aceitos")
}

File: Python/1035.py
lista = (input().split(" "))
A,B,C,D = lista
if(int(B) > int(C) and int(D) > int(A) and (int(C)+int(D)) > (int(A)+int(B)) and int(C) > 0 and int(D) > 0 and int(A)%2==0):
    print("Valores aceitos")
else:
    print("Valores nao aceitos")
